pkgs/development/tools/java/jprofiler/artefacts.nix

```diff
@@ -0,0 +1,44 @@
+rec {
+  version = "13.0.2";
+  downloadVersion = builtins.replaceStrings ["."] ["_"] version;
+
+  agent = rec {
+    x86_64-darwin = {
+      url = "https://download-gcdn.ej-technologies.com/jprofiler/jprofiler_agent_macos_${downloadVersion}.tgz";
+      hash = "sha256-GJYDA6aNP+NYARDhQy7hUy5NduRzz1xpdP9FSWk1XMM=";
+    };
+    aarch64-darwin = x86_64-darwin;
+
+    x86_64-linux = {
+      url = "https://download-gcdn.ej-technologies.com/jprofiler/jprofiler_agent_linux-x86_${downloadVersion}.tar.gz";
+      hash = "sha256-kZ2/ScbOCxeXN++RDf14gdPFrZW8oox+xfK/VUFK+mY=";
+    };
+    i686-linux = x86_64-linux;
+    aarch64-linux = {
+      url = "https://download-gcdn.ej-technologies.com/jprofiler/jprofiler_agent_linux-arm_${downloadVersion}.tar.gz";
+      hash = "sha256-VtGAZuuhyrNL0YM5LR5NKfiIcsv10qy5n8cWlxK+IRk=";
+    };
+    armhf-linux = aarch64-linux;
+  };
+
+  ui = rec {
+    x86_64-darwin = {
+      url = "https://download-gcdn.ej-technologies.com/jprofiler/jprofiler_macos_${downloadVersion}.tgz";
+      hash = "sha256-Fc7inWD5LGenkitk4/2/xHZXfaP9shvmkYvpd/BkHNE=";
+    };
+    aarch64-darwin = x86_64-darwin;
+
+    x86_64-linux = {
+      url = "https://download-gcdn.ej-technologies.com/jprofiler/jprofiler_linux_${downloadVersion}.tar.gz";
+      hash = "sha256-x9I7l2ctquCqUymtlQpFXE6+u0Yg773qE6MvAxvCaEE=";
+    };
+    i686-linux = x86_64-linux;
+    aarch64-linux = x86_64-linux;
+    armhf-linux = x86_64-linux;
```

Comment:
Instead of aliasing systems here we should just use stdenv.isLinux and stdenv.isDarwin.

With this the file should also be inlined into the default.nix.

Reply:
@SuperSandro2000 I don't think that `stdenv.isLinux` is good idea because agent has differnet linux distirbutives